lib/identity/client.js

```diff
@@ -886,9 +1096,7 @@ class Client extends PartialClient {
       )
     }
     // if the admin sharing group will be empty, revoke the share & delete the group
-    const groupName = `tozny.secret.${this.config.realmName}.${
-      this.storage.config.clientId
-    }.${secret.meta.plain.secretType.toLowerCase()}`
+    const groupName = `tozny.secret.${this.storage.config.clientId}.${this.config.realmName}`
```

Comment:
I think RealmName should still come before clientID as namespacing is usually from general -> less general, and realm (currently) is higher order than the client id.

Reply:
Luke had wanted to switch it for admin namespace incase the user gave their clientID for a namespace there was no way of them accidentally messing with this namespace

Reply:
pending impactful examples and with the assumption that any way things could get messed with in regards to client id is the same liability for messing with things by providing realm name as part of the namespace, please change this back to

`{tozny.secret}.{realmName}.{namespace} where namespace could be clientID`